src/components/MovieList/MovieList.js

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import MovieCard from '../MovieCard/MovieCard';
+import MovieCounter from '../MovieCounter/MovieCounter';
+import './MovieList.scss';
+
+const MovieList = (props) => {
```

Comment:
lets extract props in arguments like
`const MovieList = ({ movies }) => {`

Reply:
Looks great! Updated in 2b872ed